### PR TITLE
Lysander

### PR DESCRIPTION
**Pull request by lysan0333**
This is Lysander's todolist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,4 +94,5 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [Yusuke Nishikado](./yusukenishikado/README.md)
 - [Carisno Remes](./carisno/)
 - [Andreas Fischer](./andreas-fischer/README.md)
+- [Lysander Stewart](./lysander/README.md)
```
Comment (oddcommits) [inline README.md:97]: incorrect place

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lysan0333) [reply README.md]: I have fixed my position of contributors

File: README.md [hunk 2]
```diff
@@ -95,5 +95,8 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [Carisno Remes](./carisno/)
 - [Andreas Fischer](./andreas-fischer/README.md)
 - [Nguyen Aiden](./naidendev/README.md)
+- [Lysander Stewart](./lysander/README.md)
 - [WeiJie Zhang](./weijie-zhang/README.md)
 - [Alice Ibrahimovic](./alice/README.md)
+- [Lysander Stewart](./lysander/README.md)
```
Comment (oddcommits) [inline README.md:101]: you have your illusion?

**Review by oddcommits — CHANGES_REQUESTED** [review]
you should double check before a review submission

File: README.md (hunk 2 above)
Comment (lysan0333) [reply README.md]: Excuse me, I had same mistakes as preview, anyway I was greeting about your help ,thank you

File: README.md [hunk 3]
```diff
@@ -97,3 +97,5 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [Nguyen Aiden](./naidendev/README.md)
 - [WeiJie Zhang](./weijie-zhang/README.md)
 - [Alice Ibrahimovic](./alice/README.md)
+- [Lysander Stewart](./lysander/README.md)
+
```
Comment (oddcommits) [inline README.md:101]: why new line? there's already an empty line that must be kept

**Review by oddcommits — CHANGES_REQUESTED** [review]

**Review by oddcommits — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (lysan0333) [reply README.md]: Sorry , this is my mistake but I deleted this line

**Pull request merged**